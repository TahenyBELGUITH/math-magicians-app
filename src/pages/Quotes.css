.container {
  background-color: rgb(30, 130, 30);
  border: 3px solid burlywood;
  width: 70%;
  padding: 3rem;
  color: #fff;
  margin: 7rem auto;
  line-height: 50px;
}

span {
  font-size: 20px;
  color: white;
}
