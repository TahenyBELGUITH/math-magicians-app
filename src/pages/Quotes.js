import React from 'react';
import './Quotes.css';

export default function Quotes() {
  return (
    <div className="container">
      <h3>
        Without mathematics, there’s nothing you can do. Everything around you
        is mathematics. Everything around you is numbers. —
        {' '}
        <span>Shakuntala Devi</span>
        {' '}
        —
      </h3>
    </div>
  );
}
