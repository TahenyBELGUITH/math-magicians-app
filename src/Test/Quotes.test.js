import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../pages/Quotes';

it('it should match the quotes page', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
