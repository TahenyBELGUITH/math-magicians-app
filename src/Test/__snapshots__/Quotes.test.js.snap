// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should match the quotes page 1`] = `
<div
  className="container"
>
  <h3>
    Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers. —
     
    <span>
      Shakuntala Devi
    </span>
     
    —
  </h3>
</div>
`;
